### PR TITLE
[Magic Firewall] Full packet captures

### PR DESCRIPTION
**Pull request by dcpena**
Adding docs for full packet captures in the dash to address PCX-4280.

### CODE REVIEW

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 1]
```diff
@@ -1,27 +1,69 @@
...
+</details>
+
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet capture**, select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 2]
```diff
@@ -1,27 +1,69 @@
...
+3. From the **Packet Capture** page, select the **Buckets** tab.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
3. From the **Packet captures** page, select the **Buckets** tab.
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 3]
```diff
@@ -1,27 +1,69 @@
...
+4. Select **Create bucket**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
4. Select **Add a bucket**.
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 4]
```diff
@@ -78,23 +127,58 @@ This will show the following response:
...
+
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
+
+1. From the **Prove ownership** step of the **Bucket configuration**, locate the **Ownership token** field.
+2. In the **Ownership token** field, enter the ownership token for your service provider.
+3. When you are done, select **Create**. The Packet Captures page displays.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
3. When you are done, select **Create**. The **Packet captures** page displays.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 5]
```diff
@@ -132,33 +144,32 @@ header: Simple PCAP example response
...
+</details>
+
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet capture**, select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 6]
```diff
@@ -132,33 +144,32 @@ header: Simple PCAP example response
...
+3. Select the **Captures** tab and select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
3. Select the **Captures** tab and select **Start a capture**.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 7]
```diff
@@ -132,33 +144,32 @@ header: Simple PCAP example response
...
+4. From **Packet Capture configuration**, choose a capture type and select **Next**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
4. From **Packet captures - Capture configuration**, choose a capture type and select **Next**.
```

Comment (alexchenh) [reply content/magic-firewall/how-to/collect-pcaps.md]: Handle heading + subheading

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 8]
```diff
@@ -132,33 +144,32 @@ header: Simple PCAP example response
...
+5. Fill out the required fields to begin the capture and then select **Start**.
 
-The response will have the `status` field set to `pending` while the collection is in progress. You must wait for the PCAP collection to complete before downloading the file. When the PCAP is ready to download, the status will change to `success`.
+The main **Packet Capture** page displays and displays a list of captures.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
The main **Packet captures** page displays and displays a list of captures.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 9]
```diff
@@ -197,21 +208,49 @@ header: Simple PCAP example result
...
-## Download PCAP
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
 
-Depending on the `type` of PCAP there are multiple ways to obtain your PCAP.
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet capture**, select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 10]
```diff
@@ -197,21 +208,49 @@ header: Simple PCAP example result
...
 For more information on how to process multiple saved capture files into a single output file, refer to [Wireshark's mergecap documentation](https://www.wireshark.org/docs/man-pages/mergecap.html).
 
-### Full PCAPs
+**Full PCAPs**
 
-To obtain full PCAPs, download the files from the bucket specified in `destination_conf` after the PCAP's status is `success`. There may be multiple files named `pcap_<pcap_id>.pcap` per capture as captures may occur across multiple machines.
+To obtain full PCAPs, download the files from the bucket specified in `destination_conf` after the PCAP's status is `success`. You may find multiple files named `pcap_<pcap_id>.pcap` per capture as captures can occur across multiple machines.
 
-### Simple PCAPs
+**Sample PCAPs**
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:251]:
```suggestion
**Simple PCAPs**
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 11]
```diff
@@ -197,21 +208,49 @@ header: Simple PCAP example result
...
 
-Once the Simple PCAP collection is complete, you can download the PCAP by specifying the PCAP identifier used earlier.
+Once the sample PCAP collection is complete, you can download the PCAP by specifying the PCAP identifier used earlier.
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:253]:
```suggestion
Once the simple PCAP collection is complete, you can download the PCAP by specifying the PCAP identifier used earlier.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 12]
```diff
@@ -1,60 +1,47 @@
...
+## Send a packet capture request
 
-Before starting a `full` type packet capture, you must first follow instructions for [configuring a bucket](../pcaps-bucket-setup).
+Currently, when a packet capture is requested, packets flowing at the edge through the Magic Transit system are captured, and the system is `magic-transit`.
 
-{{</Aside>}}
+Cloudflare supports two types of packet captures:
 
-## Send a PCAP collect Request
+- **Sample**: Sample packets are best for debugging and providing a global picture across all data centers. Sample packets generate single, smaller files and only contain the first 160 bytes of the payload. Sampled packets are collected across all edge metals to build a PCAP file.
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:23]:
```suggestion
- **Simple**: These packets are sampled, and are best for debugging and providing a global picture across all data centers. Sampled packets generate single, smaller files and only contain the first 160 bytes of the payload. Sampled packets are collected across all edge metals to build a PCAP file.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 13]
```diff
@@ -1,60 +1,47 @@
...
 
-- `time_limit`: The number of seconds to limit the PCAP. The number should be less than `300` seconds and cannot be set to zero.
-- `type`: Can be `full` or `simple` as described above.
-- `system`: Must be `magic-transit` as described above.
+The PCAPs API needs both `system` and `type` to be specified in order to start a capture. A PCAP's `system` is the product or logical subsystem where packets are captured, and a PCAP's `type` is how the captured packets are built into a PCAP file.
 
-If a full `type` is specified the following additional fields are required:
-- `destination_conf`: This specifies the bucket path. For example, `gs://my-bucket`.
-- `colo`: This specifies the `colo` name we want to capture packets from. For example, `ord02`.
+To create a sample PCAP request, send a JSON body with the required parameter listed at [Create sample PCAP request](https://api.cloudflare.com/#magic-pcap-collection-create-simple-pcap-request).
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:34]:
```suggestion
To create a simple PCAP request, send a JSON body with the required parameter listed at [Create simple PCAP request](https://api.cloudflare.com/#magic-pcap-collection-create-simple-pcap-request).
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 14]
```diff
@@ -1,60 +1,47 @@
...
 
-Two optional fields are available to limit the amount of packets captured:
-- `packet_limit`: The number of packets to limit the PCAP. This number should be less than `10000` and cannot be set to zero. This field can be used with both `full` and `simple` PCAP types.
-- `byte_limit`: The number of bytes to limit the PCAP. This number should be less than `1000000000` bytes. This field can only be used with a `full` PCAP type.
+For full PCAP requests, refer to the required parameters listed at [Create full PCAP requests](https://api.cloudflare.com/#magic-pcap-collection-create-full-pcap-request). Note that full packet captures require two additional parameters than sample packets.
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:36]:
```suggestion
For full PCAP requests, refer to the required parameters listed at [Create full PCAP requests](https://api.cloudflare.com/#magic-pcap-collection-create-full-pcap-request). Note that full packet captures require two additional parameters than simple packets.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 15]
```diff
@@ -1,60 +1,47 @@
...
 
-In addition to the above fields, the JSON body can optionally filter packets by specifying any of
+The full PCAP request endpoint also contains optional fields you can use to limit the amount of packets captured. Both full and sample packet requests contain an optional `filter_v1` parameter you can use to filter packets by IPv4 Source address, for example. For a full list of the filter options, refer to the parameter lists above.
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:38]:
```suggestion
The full PCAP request endpoint also contains optional fields you can use to limit the amount of packets captured. Both full and simple packet requests contain an optional `filter_v1` parameter you can use to filter packets by IPv4 Source address, for example. For a full list of the filter options, refer to the parameter lists above.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 16]
```diff
@@ -1,60 +1,47 @@
...
 
-- IPv4 Source address
-- IPv4 Destination address
-- (TCP/UDP) Source port
-- (TCP/UDP) Destination port
-- IP Protocol
+Currently, you can only send one collect request per minute for sample PCAPs, and you can only have one running or pending full PCAP at a time.
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:40]:
```suggestion
Currently, you can only send one collect request per minute for simple PCAPs, and you can only have one running or pending full PCAP at a time.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 17]
```diff
@@ -76,9 +63,38 @@ header: Full PCAP example request
...
+        "byte_limit": 100000000,
+        "colo": "sfo06",
+        "destination_conf": "gs://test-magic-pcaps"
+      },
+      "success": true,
+      "errors": [],
+      "messages": []
+    }
+```
+**Sample PCAPs**
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:91]:
```suggestion
**Simple PCAPs**
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 18]
```diff
@@ -76,9 +63,38 @@ header: Full PCAP example request
...
+
+`filter_v1` can be left empty to collect all packets without any filtering.
+
 ```bash
 ---
-header: Simple PCAP example request
+header: Sample PCAP example request
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:97]:
```suggestion
header: Simple PCAP example request
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 19]
```diff
@@ -99,15 +115,11 @@ header: Simple PCAP example request
...
-
-### Example responses
-
-The response to this message will be a JSON body containing the details of the job running to build the packet capture. The response will contain a unique identifier for the packet capture request, and the details that were sent in the request.
+The response is a JSON body that contains the details of the job running to build the packet capture. The response contains a unique identifier for the packet capture request along with the details sent in the request.
 
 ```bash
 ---
-header: Simple PCAP example response
+header: Sample PCAP example response
```
Comment (alexchenhz) [inline content/magic-firewall/how-to/collect-pcaps.md:122]:
```suggestion
header: Simple PCAP example response
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 20]
```diff
@@ -111,8 +195,21 @@ The response will look like:
...
+</details>
+
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet capture**, select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/pcaps-bucket-setup.md [hunk 21]
```diff
@@ -111,8 +195,21 @@ The response will look like:
...
+3. From the **Packet Capture** page, select the **Buckets** tab.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/pcaps-bucket-setup.md]:
```suggestion
3. From the **Packet captures** page, select the **Buckets** tab.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 22]
```diff
@@ -221,20 +260,51 @@ Once the Simple PCAP collection is complete, you can download the PCAP by specif
...
+</details>
+
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet capture**, select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 23]
```diff
@@ -221,20 +260,51 @@ Once the Simple PCAP collection is complete, you can download the PCAP by specif
...
+3. From the **Packet Capture** page, select the **Captures** tab.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
3. From the **Packet captures** page, select the **Captures** tab.
```

File: content/magic-firewall/how-to/collect-pcaps.md (hunk 22 above)
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 24]
```diff
@@ -253,3 +323,21 @@ The response will include an array of up to 50 requests sent in the past and wil
...
+</details>
+
+<details>
+<summary>
+  Dashboard
+</summary>
+ <div class="special-class" markdown="1">
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet capture**, select **Start to capture**.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 25]
```diff
@@ -253,3 +323,21 @@ The response will include an array of up to 50 requests sent in the past and wil
...
+3. From the **Packet Capture** page, select the **Captures** tab.
```
Comment (alexchenh) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
3. From the **Packet captures** page, select the **Captures** tab.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 26]
```diff
@@ -1,60 +1,47 @@
...
-  title: Packet Captures (PCAPs) API
+weight: 5
 ---
 
-# Packet Captures (PCAPs) API
+# Collect packet captures (PCAPs)
 
-The PCAPs API can be used to capture packets destined for a customer origin.
-A packet capture is requested and after the capture is collected the output is contained within one or more files in PCAP capture file format.
+After a packet capture is requested and the capture is collected, the output is contained within one or more files in PCAP  file format. Before starting a `full` type packet capture, you must first follow instructions for [configuring a bucket](../pcaps-bucket-setup).
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
After a packet capture is requested and the capture is collected, the output is contained within one or more files in PCAP file format. Before starting a `full` type packet capture, you must first follow instructions for [configuring a bucket](../pcaps-bucket-setup).
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 27]
```diff
@@ -1,60 +1,47 @@
...
+Currently, when a packet capture is requested, packets flowing at the edge through the Magic Transit system are captured, and the system is `magic-transit`.
 
-{{</Aside>}}
-
-## Send a PCAP collect Request
+Cloudflare supports two types of packet captures:
 
-To send a collect request, send a JSON body specifying for all PCAP types:
+- **Simple**: Simple packets are best for debugging and providing a global picture across all data centers. Simple packets generate single, smaller files and only contain the first 160 bytes of the payload. Sampled packets are collected across all edge metals to build a PCAP file.
+- **Full**: Full packets are best for targeted data collection with a detailed view into a single data center. Full packets generate multiple large files, and they are captured within a given point-of-presence or data center and sent to either a GCP or AWS bucket specified by the user. T
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
- **Full**: Full packets are best for targeted data collection with a detailed view into a single data center. Full packets generate multiple large files, and they are captured within a given data center and sent to either a GCP or AWS bucket specified by the user.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 28]
```diff
@@ -1,60 +1,47 @@
...
+<details>
+<summary>
+  API
+</summary>
+ <div class="special-class" markdown="1">
 
-If a full `type` is specified the following additional fields are required:
-- `destination_conf`: This specifies the bucket path. For example, `gs://my-bucket`.
-- `colo`: This specifies the `colo` name we want to capture packets from. For example, `ord02`.
+The PCAPs API needs both `system` and `type` to be specified in order to start a capture. A PCAP's `system` is the product or logical subsystem where packets are captured, and a PCAP's `type` is how the captured packets are built into a PCAP file.
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
The PCAPs API needs both `system` and `type` to be specified to start a capture. A PCAP's `system` is the product or logical subsystem where packets are captured, and a PCAP's `type` is how the captured packets are built into a PCAP file.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 29]
```diff
@@ -1,60 +1,47 @@
...
 
-Two optional fields are available to limit the amount of packets captured:
-- `packet_limit`: The number of packets to limit the PCAP. This number should be less than `10000` and cannot be set to zero. This field can be used with both `full` and `simple` PCAP types.
-- `byte_limit`: The number of bytes to limit the PCAP. This number should be less than `1000000000` bytes. This field can only be used with a `full` PCAP type.
+To create a simple PCAP request, send a JSON body with the required parameter listed at [Create simple PCAP request](https://api.cloudflare.com/#magic-pcap-collection-create-simple-pcap-request).
 
-In addition to the above fields, the JSON body can optionally filter packets by specifying any of
+For full PCAP requests, refer to the required parameters listed at [Create full PCAP requests](https://api.cloudflare.com/#magic-pcap-collection-create-full-pcap-request). Note that full packet captures require two additional parameters than simple packets.
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
For full PCAP requests, refer to the required parameters listed at [Create full PCAP requests](https://api.cloudflare.com/#magic-pcap-collection-create-full-pcap-request). Note that full packet captures require two more parameters than simple packets.
```

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 30]
```diff
@@ -1,60 +1,47 @@
...
-- (TCP/UDP) Destination port
-- IP Protocol
+The full PCAP request endpoint also contains optional fields you can use to limit the amount of packets captured. Both full and simple packet requests contain an optional `filter_v1` parameter you can use to filter packets by IPv4 Source address, for example. For a full list of the filter options, refer to the parameter lists above.
 
 Currently, you can only send one collect request per minute for simple PCAPs, and you can only have one running or pending full PCAP at a time.
 
-### Example requests
+**Full PCAPs**
+
+`filter_v1` can be left empty to collect all packets without any filtering.
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]: Passive voice, consider tweaking "Leave `filter_v1` empty to collect all packets without any filtering."

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 31]
```diff
@@ -76,6 +63,35 @@ header: Full PCAP example request
...
+        "destination_conf": "gs://test-magic-pcaps"
+      },
+      "success": true,
+      "errors": [],
+      "messages": []
+    }
+```
+**Simple PCAPs**
+
+`filter_v1` can be left empty to collect all packets without any filtering.
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]: Same comment as above regarding passive voice

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 32]
```diff
@@ -132,33 +144,32 @@ header: Simple PCAP example response
...
+ <div class="special-class" markdown="1">
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
+3. Select the **Captures** tab and select **Start a capture**.
+4. From **Packet captures - Capture configuration**, choose a capture type and select **Next**.
+5. Fill out the required fields to begin the capture and then select **Start**.
 
-The response will have the `status` field set to `pending` while the collection is in progress. You must wait for the PCAP collection to complete before downloading the file. When the PCAP is ready to download, the status will change to `success`.
+The main **Packet captures** page displays and displays a list of captures.
```
Comment (crwaters16) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
The main **Packet captures** page displays a list of captures.
```

**Review by crwaters16 — APPROVED** [review]
A few nits - love the setup with API + dashboard sections!

File: content/magic-firewall/how-to/collect-pcaps.md (hunk 30 above)
Comment (dcpena) [reply content/magic-firewall/how-to/collect-pcaps.md]: Fixed!

File: content/magic-firewall/how-to/collect-pcaps.md (hunk 31 above)
Comment (dcpena) [reply content/magic-firewall/how-to/collect-pcaps.md]: Also fixed

**Pull request merged**